packages/cctp-arb/package.json

```diff
@@ -5,11 +5,14 @@
   "scripts": {
     "codegen:mainnet": "yarn prepare:mainnet && graph codegen",
     "codegen:goerli": "yarn prepare:goerli && graph codegen",
+    "codegen:sepolia": "yarn prepare:sepolia && graph codegen",
     "postinstall": "yarn codegen:mainnet",
     "prepare:mainnet": "yarn workspace @arbitrum/subgraph-common mustache $(pwd)/../subgraph-common/config/mainnet.json $(pwd)/subgraph.template.yaml | tail -n +2 > subgraph.yaml",
     "prepare:goerli": "yarn workspace @arbitrum/subgraph-common mustache $(pwd)/../subgraph-common/config/goerli.json $(pwd)/subgraph.template.yaml | tail -n +2 > subgraph.yaml",
+    "prepare:sepolia": "yarn workspace @arbitrum/subgraph-common mustache $(pwd)/../subgraph-common/config/sepolia.json $(pwd)/subgraph.template.yaml | tail -n +2 > subgraph.yaml",
     "deploy:mainnet": "graph build && yarn prepare:mainnet && graph deploy --node https://api.thegraph.com/deploy/ chrstph-dvx/cctp-arb-one",
     "deploy:goerli": "graph build && yarn prepare:goerli && graph deploy --node https://api.thegraph.com/deploy/ chrstph-dvx/cctp-arb-goerli",
```

Comment:
since goerli is deprecated, do we want to remove it?

Reply:
Removed